stl/inc/chrono

```diff
@@ -2176,7 +2176,8 @@ namespace chrono {
 
     _NODISCARD inline string _Tzdb_update_version(const string_view _Version, const size_t _Num_leap_seconds) {
         string _Icu_version{_Version.substr(0, _Version.find_last_of('.'))};
-        return _STD move(_Icu_version) + "." + _STD to_string(_Num_leap_seconds);
+        (_Icu_version += '.') += _STD to_string(_Num_leap_seconds);
+        return _Icu_version;

```

Comment:
If I understand correctly, the new version dynamically allocates exactly the same time(s) as the existing one. Could you explain in the description of this pull request what is considered optimized?

----
Given that:
- changing from `"."` to `'.'` makes a simpler overload selected, and
- resuing of `_Icu_version` can reduce the number of `string` object to create,

perhaps we can say this PR can save throughput, and improve performance in debug mode.